src/main/scala/cromwell/engine/io/gcs/GcsFileAttributes.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.engine.io.gcs
+
+import java.nio.file.Path
+import java.nio.file.attribute.{FileTime, BasicFileAttributes}
+
+class GcsFileAttributes(path: Path) extends BasicFileAttributes {
+
```

Comment:
What message should we provide ? :/ This can be implemented (it's not an `UnsupportedOperationException`), it just hasn't been yet because we don't need it for now. I think `new NotImplementedError()` carries all the information already no ?